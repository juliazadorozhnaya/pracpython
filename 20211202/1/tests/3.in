class C(metaclass=dump):
    param1 = 'nothing'
    param2 = 'else'

    def get(self, val, seq=[]):
        if len(seq) > 0:
            return [*seq, val]
        return val

    def __setitem__(self, name, val):
        pass

    def __getitem__(self, name):
        return False

a = C()
print(a.get(10))
print(a.get(10, '3854375849'))
a['hi'] = 10
print(a['hi'])
