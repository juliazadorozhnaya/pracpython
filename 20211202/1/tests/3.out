get: (10,), {}
10
get: (10, '3854375849'), {}
['3', '8', '5', '4', '3', '7', '5', '8', '4', '9', 10]
__setitem__: ('hi', 10), {}
__getitem__: ('hi',), {}
False
